torch_xla/experimental/xla_sharding.py

```diff
@@ -70,6 +71,128 @@ def get_logical_mesh(self):
     return self.device_ids.reshape(self.mesh_shape)
 
 
+class HybridMesh(Mesh):
+  ici_mesh_shape: Tuple[int, ...]
+  dcn_mesh_shape: Tuple[int, ...]
+
+  def __init__(self,
+               device_ids: Union[np.ndarray, List],
```

Comment:
Like we discussed offline, let's look into removing `device_ids` as a parameter here and defaulting to all global devices - it looks like right now we're silently ignoring the user's input in the hybrid_mesh case

cc @alanwaketan

Reply:
May I ask why, it looks like Mesh takes the device_ids as well.

Reply:
@jonb377 and I discussed offline. I got what Jon means.